packages/react/src/global-nav-profile-menu/GlobalNavProfileMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ReactNode } from "react";
+
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import { IconEllipsis } from "../icons/IconEllipsis";
+import { Text } from "../text";
+
+export type GlobalNavProfileMenuProps = BoxProps<
+  "div",
+  {
+    avatar: ReactNode;
+    name?: string;
+    organization?: string;
+  }
+>;
+
+export const GlobalNavProfileMenu = ({
+  avatar,
+  name,
+  organization,
+}: GlobalNavProfileMenuProps) => {
+  return (
+    <Flex alignItems="center" flexDirection="row" gap="xs" px="md" py="xs">
+      <Box asChild rounded="sm" size="md">
```

Comment:
We need to use avatar here and use a `sm` radius for it

Reply:
If we're passing `Avatar` as a Reactnode, I think handling the rounded part should be done there. 
Or we can just pass the img src and handle it by wrapping it with the `Avatar` component here.

C.c - @radnan bhai, what do you think?

Reply:
i think this is one exception to our general rule here - so we'll use Avatar and explicitly pass `rounded` to get the shape we want

maybe in the future we can add this as a variant to the Avatar component - but it's not needed right now